Sampled/UI/VIews/CombinedPlayerView.swift

```diff
@@ -0,0 +1,260 @@
+          }
+          
+          HStack(spacing: 4) {
+            switch track.artistName {
+              case .empty:
+                EmptyView()
+              case .value(let artist):
+                Text(artist ?? "Unknown Artist")
+                  .font(.subheadline)
+                  .foregroundStyle(.secondary)
+                  .lineLimit(1)
+              case .mixed:
+                Text("Various Artists")
+                  .font(.subheadline)
+                  .foregroundStyle(.tertiary)
+                  .lineLimit(1)
+            }
+            
+            if case .value(let albumName) = track.albumName, let name = albumName {
+              Text("•")
+                .foregroundStyle(.quaternary)
+              Text(name)
+                .font(.subheadline)
+                .foregroundStyle(.secondary)
+                .lineLimit(1)
+            }
+          }
+        }
+        .frame(width: 250, alignment: .leading)
+        
+        Spacer()
+        
+        // Playback controls (centered)
+        HStack(spacing: 20) {
+          Button(action: onPrevious) {
+            Image(systemName: "backward.fill")
+              .font(.title3)
+          }
+          .buttonStyle(.plain)
+          .help("Previous track")
+          
+          Button(action: onPlayTapped) {
+            Image(systemName: isPlaying ? "pause.circle.fill" : "play.circle.fill")
+              .font(.system(size: 36))
+          }
+          .buttonStyle(.plain)
+          .help(isPlaying ? "Pause" : "Play")
+          
+          Button(action: onNext) {
+            Image(systemName: "forward.fill")
+              .font(.title3)
+          }
+          .buttonStyle(.plain)
+          .help("Next track")
+        }
+        .frame(maxWidth: 200)
+        
+        Spacer()
+        
+        // Duration and controls
+        HStack(spacing: 12) {
+          switch track.duration {
+            case .empty:
+              EmptyView()
+            case .value(let duration):
+              LibraryTrackDurationView(duration: duration)
+                .font(.caption)
+                .foregroundStyle(.secondary)
+            case .mixed:
+              if let total = track.totalDuration, let average = track.averageDuration {
+                VStack(alignment: .trailing, spacing: 2) {
+                  HStack(spacing: 4) {
+                    Text("Total:")
+                      .foregroundStyle(.tertiary)
+                    LibraryTrackDurationView(duration: total)
+                  }
+                  HStack(spacing: 4) {
+                    Text("Avg:")
+                      .foregroundStyle(.tertiary)
+                    LibraryTrackDurationView(duration: average)
+                  }
+                }
+                .font(.caption)
+                .foregroundStyle(.secondary)
+              }
+          }
+          
+          Divider()
+            .frame(height: 20)
+          
+          Button(action: {}) {
```

Comment:
If the button takes no action, it probably shouldn't have a button style (it'll mess up accessibility). I believe `.plain` draws no background, so you may need to set prominence, instead.